manifests/09-prometheus-rules.yaml

```
@@ -28,3 +28,18 @@ spec:
           DeploymentConfigs which reference ImageStreamTags may not work as
           expected. Please configure storage and update the config to Managed
           state by editing configs.imageregistry.operator.openshift.io.
+  - name: ImagePruner
+    rules:
+    - alert: ImagePruningDisabled
+      expr: image_registry_operator_image_pruner_install_info{} < 2
```

You don't need the empty `{}`. 
```suggestion
      expr: image_registry_operator_image_pruner_install_info < 2
```